org.speed_dreams.SpeedDreams.yaml

```diff
@@ -5,7 +5,6 @@ sdk: org.freedesktop.Sdk
 command: speed-dreams-2
 
 finish-args:
-  - --socket=wayland
```

Comment:
you can't remove this and keep `fallback-x11`. it must be replaced by `x11`

Reply:
Updated the commit.